tests/inspection/test_info.py

```diff
@@ -131,6 +131,27 @@ def test_info_from_poetry_directory():
     demo_check_info(info)
 
 
+def test_info_from_poetry_directory_fallback_on_poetry_create_error(
+    mocker: MockerFixture,
+):
+    import poetry.inspection.info
+
+    mocker.patch(
+        "poetry.inspection.info.Factory.create_poetry", side_effect=RuntimeError
+    )
+    mock_get_poetry_package = mocker.spy(
+        poetry.inspection.info.PackageInfo, "_get_poetry_package"
```

Comment:
```suggestion
        PackageInfo, "_get_poetry_package"
```

Can't we just use `PackageInfo` so that the local import is not required?

Reply:
Done